packages/migrate/src/__tests__/MigrateDiff.test.ts

```diff
@@ -508,7 +508,7 @@ describe('migrate diff', () => {
         CREATE TABLE [dbo].[Blog] (
             [id] INT NOT NULL,
             [viewCount20] INT NOT NULL,
-            CONSTRAINT [Blog_pkey] PRIMARY KEY ([id])
+            CONSTRAINT [Blog_pkey] PRIMARY KEY CLUSTERED ([id])
```

Comment:
Uh, is that one really expected?

Reply:
To double check with @pimeys and @tomhoule I think it's the same change as I can see in the tests here https://github.com/prisma/prisma-engines/pull/2861/files#diff-7afb0491c9735af619fae3163f036684b8c555949b5bb3d5a55faaa3015a0bd0

Note the schema for this snapshot is 
```
datasource my_db {
  provider = "sqlserver"
  url      = env("TEST_MSSQL_JDBC_URI_MIGRATE")
}

model Blog {
  id          Int @id
  viewCount20 Int
}
```